src/scipp/utils/to_string.py

```diff
@@ -23,12 +23,17 @@ def name_with_unit(var=None, name=None, log=False):
     return text
 
 
-def value_to_string(val, precision=3):
+def value_to_string(val, precision=3, max_str_len=80):
     """
     Convert a number to a human readable string.
     """
     if (not isinstance(val, float)) or (val == 0):
-        text = str(val)
+        raw_text = str(val)
+        if len(raw_text) > max_str_len:
+            preview_len = max_str_len - 3
+            text = f"{raw_text[:preview_len]}..."
```

Comment:
Should this be
```suggestion
            text = f"{raw_text[:preview_len-3]}..."
```
or something like that?

Reply:
I applied the change and removed the `preview_len` variable